RetailSuiteAccounting/UserInterface/02a_DebtorsCreditors.md

```diff
@@ -0,0 +1,2394 @@
+  Enter the start and end date of a specific period for the statistics. Alternatively, use the ![Calendar](../../Assets/Icons/Calendar.png "[Calendar]") (Calendar) button to select the desired dates.
+
+  - *Product group*  
+  Click the drop-down list to select a specific product group.
+
+  - *Customer*  
+  Click the drop-down list to select a specific customer.
+
+
+- [+ COMPARISON PERIOD]  
+Click this button to add a comparison period in the statistics. Additional year and period selection drop-down lists are displayed to specify the comparison period.
+
+- [SHOW]  
+Click this button to display the statistics in the list.
+
+
+- [PRINT]  
+Click this button to print the statistics displayed. The *Print view* window is displayed.
+
+[comment]: <> (Fenster beschreiben?)
+
+- [EXCEL]  
+Click this button to download the statistics displayed in Excel format. By default, the downloaded export file is saved in the *Downloads* folder of your device.
+
+
+- [NEW/RESET]  
+Click this button to reset the *Customer/supplier* tab and create a new customer/supplier. The *Create customer/supplier* tab is displayed.
+
+- [DELETE CUSTOMER 'number']/[DELETE SUPPLIER 'number']  
+Click this button to delete the selected customer/supplier.
+
+- [SAVE]  
+Click this button to save all changes in the selected customer/supplier.
+
+
+### Drop shipping
+
+*Accounting > Settings > Tab DEBTORS/CREDITORS > Tab Customer/supplier list > Select supplier > Sub-tab Drop shipping*
+
+> [Info] This sub-tab is only displayed when selecting a supplier in the *Customer/supplier* list.
+
+![Delivery addresses](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/DropShipping02.png "[Delivery addresses]")
+
+[comment]: <> (HG: Fenster geschnitten)
+
+
+**Settings**
+
+- *Drop shipping*  
+The left side menu displays the drop shipping setting options. The following menu entries are available:
+
+  - *Shipper*  
+  Click this menu entry to display the *Shipping* settings.
+
+  - *Export*  
+  Click this menu entry to display the *Export* settings.
+
+  - *Receipt export*  
+  Click this menu entry to display the *Receipt export* settings.
+
+  - *Reimport*  
+  Click this menu entry to display the *Reimport* settings.
+
+
+*Shipper*
+
+![Shipper](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/DropShippingShipper.png "[Shipper]")
+
+**Shipping**
+
+- *Shipping provider*  
+Click the drop-down list to select a shipping provider. All available shipping providers are displayed in the list. Depending on the shipping provider selected, the fields displayed in the *Services* section vary.  
+
+  > [Info] The shipping provider can be configured in the *Shipping provider* tab of the *Settings* menu entry in the *Order management* module.
+
+**Services - (shipping provider name)**  
+Configure the services as agreed with the shipping provider. Depending on the shipping provider selected, the services, and therefore the displayed fields, vary.
```

Comment:
Wir müssen überlegen, wie wir das beschreiben. Auch die Felder in den übrigen Menüpunkten, z.B. Export, sind von der Wahl des Shipping providers abhängig bzw. unterscheiden sich...